lab_5_scrapper/scrapper.py

```diff
@@ -94,39 +102,33 @@ def _validate_config_content(self) -> None:
         Ensure configuration parameters
         are not corrupt
         """
-        with open(self.path_to_config, 'r', encoding='utf-8') as f:
-            content = json.load(f)
 
-        seed_urls = content['seed_urls']
-        headers = content['headers']
-        total_articles_to_find_and_parse = content['total_articles_to_find_and_parse']
-        encoding = content['encoding']
-        timeout = content['timeout']
-        should_verify_certificate = content['should_verify_certificate']
-
-        if not isinstance(seed_urls, list)\
-                or not not all(isinstance(url, str) for url in seed_urls)\
-                or not all(re.match('https?://w?w?w?.', url) for url in seed_urls):
+        if not isinstance(self._seed_urls, list)\
+                or not not all(isinstance(url, str) for url in self._seed_urls)\
+                or not all(re.match('https?://w?w?w?.', url) for url in self._seed_urls) or len(self._seed_urls) == 0:
             raise IncorrectSeedURLError
 
-        if not isinstance(headers, dict):
+        if not isinstance(self._headers, dict):
             raise IncorrectHeadersError
 
-        if not isinstance(encoding, str):
+        if not isinstance(self._encoding, str):
             raise IncorrectEncodingError
 
-        if not isinstance(total_articles_to_find_and_parse, int):
+        if not isinstance(self._num_articles, int):
             raise IncorrectNumberOfArticlesError
 
-        if total_articles_to_find_and_parse not in range(1, 151):
+        if self._num_articles not in range(1, NUM_ARTICLES_UPPER_LIMIT + 1):
             raise NumberOfArticlesOutOfRangeError
 
-        if not isinstance(timeout, int) or not timeout < 60 or not timeout > 0:
+        if not isinstance(self._timeout, int) or self._timeout not in range(TIMEOUT_LOWER_LIMIT, TIMEOUT_UPPER_LIMIT + 1):
             raise IncorrectTimeoutError
 
-        if not isinstance(should_verify_certificate, bool):
+        if not isinstance(self._verify_certificate, bool):
             raise IncorrectVerifyError
 
+        if not isinstance(self._headless_mode, bool):
+            raise IncorrectHeadlessError
```

Comment:
According to task description and tests, `IncorrectVerifyError` should be thrown in this case

Reply:
done